backend/python_init_script/main.py

```diff
@@ -2,41 +2,186 @@
 # SPDX-FileCopyrightText: 2021 Felix Steinkohl <steinkohl@campus.tu-berlin.de>
 
 import os
+from dataclasses import dataclass
+import pandas as pd
 from appwrite.client import Client
 from appwrite.services.users import Users
 from appwrite.services.teams import Teams
+from appwrite.exception import AppwriteException
+
+
+@dataclass
+class UserData:
+    """
+    Dataclass for specifying user data
+    """
+
+    name: str
+    email: str
+    team: str
+    roles: [str]
+    init_pw: str
+
+
+def load_user_data(file: str) -> [UserData]:
+    """
+    Reads the given file and parses it into UserData objects.
+
+    :param file: path of file containing the user data that shall be loaded
+    :type file: str
+    :return: iterable of the loaded UserData objects
+    :rtype: [UserData]
+    """
+    ending = str(file).lower().split(".")[-1]
+    user_df = pd.DataFrame()
+    if ending == "csv":
+        user_df = pd.read_csv(file, dtype=str)
+        # if there are less than 5 columns -> there might be ';' used as separators
```

Comment:
Maybe a poor comment. What about
```
column_count = user_df.shape[1]
if column_count < 5: ...
```

Reply:
here i don't see the improvement, the comment is all about why 5 columns is a messure

Reply:
Wouldn't you say, it is redundant? Because it tries to explain code. I think your code explains itself well. To make clear that it's about columns you could use `column_count` as name. So after all, variable names just are comments which explain what something does.